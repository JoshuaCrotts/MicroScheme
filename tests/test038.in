(define (equals num)
    (cond
        [(= num 5) "Num is 5"]
        [(< num 5) "Num is less than 5"]
        [else "Num > 5"]))

(equals 5)
(equals 6)
(equals 4)
