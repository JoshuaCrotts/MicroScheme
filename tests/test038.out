"Num is 5"
"Num > 5"
"Num is less than 5"
